language/en-GB/mod_articles.ini

```
@@ -0,0 +1,99 @@
+MOD_ARTICLES_FIELD_CATFILTERINGTYPE_DESC="Include or exclude the selected categories."
+MOD_ARTICLES_FIELD_CATFILTERINGTYPE_LABEL="Category Filtering Type"
+MOD_ARTICLES_FIELD_COUNT_DESC="The number of items to display. The default value of 0 will display all articles."
+MOD_ARTICLES_FIELD_COUNT_LABEL="Articles to Display"
+MOD_ARTICLES_FIELD_DATEFIELDFORMAT_DESC="Please enter in a valid date format. See: https://php.net/date for formatting information."
+MOD_ARTICLES_FIELD_DATEFIELDFORMAT_LABEL="Date Format"
+MOD_ARTICLES_FIELD_DATEFIELD_LABEL="Date Field"
+MOD_ARTICLES_FIELD_DATEFILTERING_LABEL="Date Filtering"
+MOD_ARTICLES_FIELD_DATEGROUPINGFIELD_DESC="Select which date field you want the date grouping to be applied to."
+MOD_ARTICLES_FIELD_DATEGROUPINGFIELD_LABEL="Date Grouping Field"
+MOD_ARTICLES_FIELD_DATERANGEFIELD_LABEL="Date Range Field"
+MOD_ARTICLES_FIELD_ENDDATE_LABEL="To Date"
+MOD_ARTICLES_FIELD_EXCLUDEDARTICLES_LABEL="Article IDs to Exclude"
+MOD_ARTICLES_FIELD_EXCLUDE_CURRENT_LABEL="Exclude Current Article"
+MOD_ARTICLES_FIELD_GROUP_DISPLAY_LABEL="Display Options"
+MOD_ARTICLES_FIELD_GROUP_FILTERING_LABEL="Filtering Options"
+MOD_ARTICLES_FIELD_GROUP_GROUPING_LABEL="Grouping Options"
+MOD_ARTICLES_FIELD_GROUP_ORDERING_LABEL="Ordering Options"
+MOD_ARTICLES_FIELD_IMAGES_ARTICLE_LABEL="Show Intro/Full Image"
+MOD_ARTICLES_FIELD_IMAGES_DESC="Show the images that are inside the text of the article."
+MOD_ARTICLES_FIELD_IMAGES_LABEL="Show Article Images"
+MOD_ARTICLES_FIELD_INCLUDEDARTICLES_LABEL="Article IDs to Include"
+MOD_ARTICLES_FIELD_INFOLAYOUT_DESC="Layout for the Articles Info (Date, Hits, Category, Author)."
+MOD_ARTICLES_FIELD_INFOLAYOUT_LABEL="Articles Info Layout"
+MOD_ARTICLES_FIELD_INTROTEXTLIMIT_DESC="Truncate Introtext to a specified number of characters. Set to 0 to disable truncation."
+MOD_ARTICLES_FIELD_INTROTEXTLIMIT_LABEL="Introtext Limit (characters)"
+MOD_ARTICLES_FIELD_LAYOUT_COL_LABEL="Max. Number of Columns"
+MOD_ARTICLES_FIELD_LAYOUT_DESC="Display articles vertical or horizontal."
+MOD_ARTICLES_FIELD_LAYOUT_HOR="Horizontal"
+MOD_ARTICLES_FIELD_LAYOUT_LABEL="Layout"
+MOD_ARTICLES_FIELD_LAYOUT_MULTILINE="Multiline"
+MOD_ARTICLES_FIELD_LAYOUT_SINGLELINE="Single Line"
+MOD_ARTICLES_FIELD_LAYOUT_VERT="Vertical"
+MOD_ARTICLES_FIELD_LINKCARD_DESC="The whole article will be linked."
+MOD_ARTICLES_FIELD_LINKCARD_LABEL="Card Style"
+MOD_ARTICLES_FIELD_LINKTITLES_LABEL="Title Link"
+MOD_ARTICLES_FIELD_MONTHYEARFORMAT_DESC="Please enter in a valid date format. See: https://php.net/date for formatting information."
+MOD_ARTICLES_FIELD_MONTHYEARFORMAT_LABEL="Month and Year Display Format"
+MOD_ARTICLES_FIELD_ONLYARCHIVED_LABEL="Archived Articles"
+MOD_ARTICLES_FIELD_RELATIVEDATE_LABEL="Relative Date"
+MOD_ARTICLES_FIELD_SHOWCATEGORYLINK_LABEL="Category Link"
+MOD_ARTICLES_FIELD_SHOWCHILDCATEGORYARTICLES_DESC="Include or exclude artucles in the child cetegories of the selected ones."
```

```suggestion
MOD_ARTICLES_FIELD_SHOWCHILDCATEGORYARTICLES_DESC="Include or exclude articles in the child categories of the selected ones."
```

spelling corrections